platform/linux-generic/odp_packet_io.c

```diff
@@ -2869,7 +2990,30 @@ int odp_pktout_send(odp_pktout_queue_t queue, const odp_packet_t packets[],
 	if (_ODP_PCAPNG)
 		_odp_dump_pcapng_pkts(entry, queue.index, packets, num);
 
-	return entry->s.ops->send(entry, queue.index, packets, num);
+	if (odp_unlikely(_odp_pktio_tx_compl_enabled(entry))) {
+		for (int i = 0; i < num; i++)
+			check_tx_compl_ev(packet_hdr(packets[i]), i, tx_compl_info, &num_tx_cevs);
+
+		if (num_tx_cevs > 0) {
```

Comment:
This could be also odp_unlikely(num_tx_cevs), as tx_compl requests should be rare.

Reply:
Done in v7.